scripts/tasks/src/verify-packaging.ts

```diff
@@ -57,12 +59,15 @@ export function verifyPackaging(options: Options) {
     0,
     `wont ship non production code related folders/files`,
   );
-  assert.equal(micromatch(processedResultArr, rootConfigFiles).length, 0, `wont ship configuration files`);
   assert.ok(micromatch(processedResultArr, 'CHANGELOG.md').length, 'ships changelog markdown file');
   assert.ok(micromatch(processedResultArr, 'dist/*.d.ts').length, 'ships rolluped dts');
   assert.ok(micromatch(processedResultArr, 'lib-commonjs/**/*.(js|map)').length, 'ships cjs');
   assert.equal(micromatch(processedResultArr, 'src/*').length, 0, `wont ship source code from "/src"`);
 
+  if (!isV8package) {
+    assert.equal(micromatch(processedResultArr, rootConfigFiles).length, 0, `wont ship configuration files`);
```

Comment:
v8 ships lot of stuff to npm which it should not - we don't have context why, so lets keep it as is.